osh/hub/scripts/osh-retention

```diff
@@ -26,96 +25,86 @@ from osh.hub.scan.models import TaskResultsRemoval  # noqa: E402
 
 logger = logging.getLogger("osh.hub.scripts.osh-retention")
 
-tasks_to_delete = []
-today = date.today()
 
+# Functions to determine membership for given retention setting
 
-def delete_tasks_results(task_list):
-    for task, retention_policy_setting in task_list:
-        try:
-            path_to_delete = Task.get_task_dir(task.id)
-            # if directory exists, we delete it. if not, we mark it as deleted as
-            # the directory already doesn't exist
-            if os.path.isdir(path_to_delete):
-                shutil.rmtree(path_to_delete)
-                logger.debug('The following directory was deleted: %s', path_to_delete)
-        except OSError as e:
-            logger.error("Error deleting task's directory: %s", e)
-            continue
+def is_failed(task):
+    return task.is_failed()
 
-        # if the tasks results are deleted, we update the database field,
-        # so we don't try to delete them again.
-        TaskResultsRemoval.objects.create(task=task, reason=retention_policy_setting)
 
+def is_personal(task):
+    # If the username does not contain /, @ or -, assume that the task is personal.
+    return not re.search('/@-', task.owner.username)
```

Comment:
do you intend to use `[/@-]` here?

Reply:
@rhyw Good catch!  This would not work as expected.

Reply:
Thank you, @rhyw!  Addressed in 555753918e1de3fe21e39893db930de5dd5001e2.